xhttp/identity/ctx.go

```diff
@@ -69,6 +69,12 @@ func SetGlobalIdentityMapper(e Mapper) {
 	identityMapper = e
 }
 
+//FromContext extracts the RequestContext stored inside a go context. Returns null if no such value exists.
+func FromContext(ctx context.Context) *RequestContext {
```

Comment:
can you pls add this call to the unittest?

Reply:
I added a test. Please take a look.